### DISCUSSION TITLE
Do How apply behaver ctrl + shift

### DISCUSSION THREAD

**Discussion by oopsfix** [dolanmiu/docx#953] [Q&A]
To paragraph applied justified, how apply behaver shift + enter at need a place?

**Comment by dolanmiu** [comment]
set the paragraph alignment to justified:

```ts

paragraph: {
    alignment: AlignmentType.JUSTIFIED,
},
```

**Reply by oopsfix** [reply]
It is applied. Need at some point paste shift + enter

**Reply by dolanmiu** [reply]
What does point paste shift enter do?

I am on Microsoft Word right now, and trying to do a combination of this, and can't figure it out

**Reply by oopsfix** [reply]
When justified is applied to a paragraph and you need to force the words shift + enter does it competently
